README.md

```diff
@@ -207,6 +239,7 @@ for more detailed instruction on app definitions.
 |-----------------|---|
 |2.0|optional|
 |3.0|optional **[Deprecated]**|
```

Comment:
A lot of 3.0 packages still use command.json 
what does deprecated + optional mean here  ?